app/models/game.rb

```diff
@@ -17,13 +17,24 @@ class Game < ActiveRecord::Base
 
   ST_LIMIT_COUNT = 150
 
+  enum status: [ :active, :culled, :storage ]
+
   def format_member_variables
     self.barcode.upcase!
   end
 
   def self.get(barcode)
     # search for game based on barcode and event
-    where(barcode: barcode.upcase, culled: false).first
+    where(barcode: barcode.upcase, status: Game.statuses[:active]).first
+  end
+
+  def self.get_stored(barcode)
```

Comment:
Might want to name this something else, very confusing.

Reply:
Changed to get_if_stored.